examples/graphql-caching-global/graphql_caching_global.md

```diff
@@ -0,0 +1,23 @@
+# GraphQL service - Operational Level Caching
```

Comment:
@praneesha What should be the recommended way here? I guess we need to use a hyphen (server-side) here.

```suggestion
# GraphQL service - Operation-level caching
```

Reply:
Yes, we need the hyphen here.

Reply:
Decided to go on with `operation-level caching`.